app/src/main/java/sdp/moneyrun/game/Game.java

```diff
@@ -209,6 +220,16 @@ public static Game getGameFromTaskSnapshot(Task<DataSnapshot> task){
         }
     }
 
+    public void addCoinListener(ValueEventListener listener){
+        if (listener == null) throw new IllegalArgumentException();
+        FirebaseDatabase.getInstance().getReference().child("open_games").child("coins").addValueEventListener(listener);
```

Comment:
I think youre on the wrong node. Did you intend to go into [...] .child("open_games").child(*GAMEID*).child("coins") ?
Or are the coins situated in the root "open_games" branch, and not in a Game in particular ?

Reply:
Actually Arnau changed it from "open_games" to just "games" so you should change that too

Reply:
Thanks for the review you were right